lambda/rotator-snowflake/rotate/rotate.go

```diff
@@ -17,8 +18,8 @@ func buildSnowflakeSecrets(ctx context.Context, snowflakeAccount *snowflakeConfi
 	}
 	snowflakeDB := snowflakeAccount.DB
 
-	userQuery := "SHOW USERS LIKE '?'"
-	connectionRow := snowflake.QueryRow(ctx, snowflakeDB, userQuery, username)
+	userQuery := fmt.Sprintf("SHOW USERS LIKE '%s'", username)
+	connectionRow := snowflake.QueryRow(ctx, snowflakeDB, userQuery)
```

Comment:
Biggest change here: we don't prep the query. We just format it directly & send it to Snowflake